src/test/data/JsonSerializableAddressBookTest/typicalClientsAddressBook.json

```diff
@@ -17,7 +26,16 @@
     "country" : "SG",
     "timezone": "GMT+8",
     "contractExpiryDate": "3-12-2022",
-    "tagged" : [ "owesMoney", "friends" ]
+    "tagged" : [ "owesMoney", "friends" ],
+    "clientNotes" : [ {
+      "contents" : "his son is the devil's incarnate",
+      "countryCode" : "NULL_CC",
+      "tags" : [ "turnOff" ]
+    }, {
+      "contents" : "snores a lot",
+      "countryCode" : "NULL_CC",
+      "tags" : [ "turnOff" ]
+    } ]
```

Comment:
no space between `[` and `{`

Reply:
does this look better? tried the autoformatting and it makes things into a pretty tree-like format

![image](https://user-images.githubusercontent.com/38996397/97250269-8e576d80-1840-11eb-85ce-35aec5243a32.png)